src/inflators/model.js

```diff
@@ -1,17 +1,55 @@
 import { addComponent, addEntity } from "bitecs";
 import { GLTFModel } from "../bit-components";
 import { loadModel } from "../components/gltf-model-plus";
-import { addObject3DComponent } from "../utils/jsx-entity";
-
-export function inflateModel(world, eid, { src }) {
-  const obj = new THREE.Group();
-  addObject3DComponent(world, eid, obj);
-  loadModel(src, null, true).then(gltf => {
-    // TODO inflating components and create animation mixers
-    const modelEid = addEntity(world);
-    addObject3DComponent(world, modelEid, gltf.scene);
-    addComponent(world, GLTFModel, modelEid);
+import { addObject3DComponent, inflators } from "../utils/jsx-entity";
+
+export function inflateModel(world, rootEid, { src }) {
```

Comment:
There is a bunch of duplicate code between this and `loadModel` (the utils one not the gltf-model-plus one), we need to review them and see which one is correct. I think the code probably belongs in this inflator though, and this should maybe take a loaded gltf instead of a src (similar to how image infiltrator takes a texture). Doesn't really map cleanly to a gltf in terms of semantics, but maybe a `model` component doesn't make sense to use in a gltf? Maybe it would be a reference to another node or scene?